app/views/docs/rest.phtml

```diff
@@ -1,12 +1,310 @@
+x-appwrite-id: 6369b0bc1dcf4ff59051
+
+------WebKitFormBoundarye0m6iNBQNHlzTpVM
+Content-Disposition: form-data; name="fileId"
+
+unique()
+------WebKitFormBoundarye0m6iNBQNHlzTpVM
+Content-Disposition: form-data; name="file"; filename="file.txt"
+Content-Type: application/octet-stream
+
+[CHUNKED-DATA]
+------WebKitFormBoundarye0m6iNBQNHlzTpVM
+Content-Disposition: form-data; name="permissions"
+
+["read(\"user:627a958ded6424a98a9f\")"]
+------WebKitFormBoundarye0m6iNBQNHlzTpVM--</code></pre>
+</div>
+
+<h2>Permissions</h2>
+
+<p>Appwrite SDKs have helpers to generate permission strings, but when using Appwrite without SDKs, you'd need to create the strings yourself.</p>
+
+<h3>Permission Types</h3>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 250px">SDK</td>
+        <td>Permission String</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>Permission.read()	</code></td>
+        <td><code>"read(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.create()</code></td>
+        <td><code>"create(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.update()</code></td>
+        <td><code>"update(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.delete()</td>
+        <td><code>"delete(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.write()</td>
+        <td><code>"write(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    </tbody>
+</table>
+
+<h3>Permission Roles</h3>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 250px">SDK</td>
+        <td>Role String</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>Role.any()</code></td>
+        <td><code>"any"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.guests()</code></td>
+        <td><code>"user:[ID]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.users([STATUS])</code></td>
+        <td><code>"users/[STATUS]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.user([USER_ID], [STATUS])</td>
+        <td><code>"user:[USER_ID]/[STATUS]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.team([TEAM_ID])</td>
+        <td><code>"team:[TEAM_ID]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.team([TEAM_ID], [ROLE])</td>
+        <td><code>"team:[TEAM_ID]/[ROLE]"</code></td>
+    </tr>
+    </tbody>
+</table>
+
+<ul>
+    <li class="margin-bottom"><a href="/docs/permissions" rel="noopener"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> Learn more about permissions</a></li>
+</ul>
+
+<h2>Unique ID</h2>
+<p>
+Appwrite's SDKs have a helper <code>ID.unqiue()</code> to generate unique IDs. When using Appwrite without an SDK, pass the string <code>"unique()"</code> into the ID parameter.
+</p>
+
+<h2>Query Methods</h2>
```

Comment:
Another thing people may not be aware of is how to pass multiple queries in 1 HTTP request. An example might help with that.

Reply:
This is a good point. On it.

Reply:
has an example been added anywhere?